src/arrayMethodSort.js

```diff
@@ -4,8 +4,26 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) > String(b)) {
```

Comment:
if a === b you need to return 0

Reply:
Agree
Take a look what is callback of sort return
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
Split this to the separate fucntion